src/cosl/coordinated_workers/nginx.py

```diff
@@ -205,3 +206,15 @@ def layer(self) -> pebble.Layer:
                 },
             }
         )
+
+
+def is_ipv6_enabled() -> bool:
+    """Check if IPv6 is enabled on the container's network interfaces."""
+    exit_code, output = subprocess.getstatusoutput("ip -6 address show")
+    if exit_code != 0:
+        # if running the command failed for any reason, assume ipv6 is not enabled.
+        logger.warning(f"Disabling ipv6 in nginx config: {output}")
```

Comment:
I think it's up to the caller to log this. this function doesn't know why it's being called.